### ISSUE TITLE
Unable to render multiday events

### ISSUE THREAD

**Issue by krishmana** [themouette/jquery-week-calendar#81]
While using this plugins json example. I gave a new start and end date. And also make necessary changes in the calendar call like date, minDate and maxDate. But the plugin is unable to render the event. Here is the json object I was trying to render.

[
    {"id":1, "start":"2011-06-04T13:15:00.000", "end":"2011-06-06T14:15:00.000", "title":"Lunch with Mike"}
]

Can you please point out the mistake I am making.

**Comment by dittodhole** [comment]
for reproducing this, someone will need your config either :)

**Comment by robotdan** [comment]
Is this the same as issue #88 ?
https://github.com/themouette/jquery-week-calendar/issues/88

**Comment by krishmana** [comment]
No, the issue is not same as #88.

The actual issue is when you have got a start date which falling on day suppose friday and then you have got a end date which falling suppose on tuesday. Then it's rendering the event till saturday only.

**Comment by krishmana** [comment]
Sorry, I closed it by mistake.

**Comment by robotdan** [comment]
Using the weekcalendar_demo_4.html, I added the following event to the events array.

<pre>
{"id":6, "start": new Date(year, month, day, 15), "end": new Date(year, month, day + 4, 17),"title":"Multi-Day"}
</pre>


This a multi day event that goes from today (Thursday in this case) to Monday.  Because this is a week calendar, it only renders until Saturday. But if I click the next button to view next week, the event does continue there until Monday as expected.

Is this what you are seeing as well?  If you want to see an event rendered past Saturday, maybe you need a full month calendar?  Or you could shift the start date of the calendar to show Wednesday to Wednesday instead of Sunday to Saturday.

Am I understanding your problem correctly?  As in #88, this does still create separate events, one for each day.

**Comment by krishmana** [comment]
Instead of adding 4 to the end_date add + 5 and see.

**Comment by robotdan** [comment]
Yes, that also works for me.  You are unable to see the event into next week if you click the next week button?

**Comment by krishmana** [comment]
yes..but it doesn't render the event in the current week also.

I don't how it's working for you....I have downloaded the code on Sep 14 2011. If there's a change then I have to check the new code.

**Comment by robotdan** [comment]
If an event spans from Thursday Oct. 27 to Tuesday Nov. 1st, and the calendar is displaying from
Sunday Oct 23 to Saturday Oct 29 - why would it display more than just Thursday to Saturday of the event?

I would expect this to work as it is, and you would need to click 'next week' on the calendar button to view the next week - which would be Sunday Oct 30 to Saturday Nov 5 to view the remainder of the scheduled event which ends on Tuesday Nov. 1st.

I'm doing this in the demo 4 and haven't changed anything else, maybe your min and max date settings are different?
